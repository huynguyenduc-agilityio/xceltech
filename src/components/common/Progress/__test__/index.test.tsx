import { render } from '@testing-library/react';

import { Progress } from '..';

describe('Progress component', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Progress
        value={30}
        indicatorContent={
          <div className="flex items-center justify-between">
            <span className="font-lg">Annual Leave</span>
            <span className="font-lg">10 of 60 day(s)</span>
          </div>
        }
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
