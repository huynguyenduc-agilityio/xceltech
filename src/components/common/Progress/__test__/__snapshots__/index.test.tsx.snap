// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress component should match snapshot 1`] = `
<div>
  <div
    class="flex flex-col flex-1 w-full"
  >
    <div
      class="flex items-center justify-between"
    >
      <span
        class="font-lg"
      >
        Annual Leave
      </span>
      <span
        class="font-lg"
      >
        10 of 60 day(s)
      </span>
    </div>
    <div
      aria-valuemax="100"
      aria-valuemin="0"
      class="relative h-7 w-full overflow-hidden rounded-[3px] bg-backgroundGray"
      data-max="100"
      data-state="indeterminate"
      role="progressbar"
    >
      <div
        class="h-full flex-1 transition-all rounded-[3px] bg-primary"
        data-max="100"
        data-state="indeterminate"
        style="transform: translateX(-70%);"
      />
    </div>
  </div>
</div>
`;
